Chapter5/stats_handler.erl

```diff
@@ -0,0 +1,13 @@
+-module(stats_handler).
```

Comment:
Its now been updated so that the handler init the Db as empty, the handler still needs to be added with "event_manager:add_handler(EventHandler, {stats_handler, InitData})." but now whatever is passed in InitData is ignored.

Are you sure its better to take away this functionality? I would have thought that allowing the alteration of the init state of the Db would be useful. For example if someone wanted to update the stats_handler module but keep the event_manager running they could read the current data in the stats_handler using get_data, close the handler, update the code and then start it with the last stat_handler's data:

OldData = event_manager:get_data(eve, stats_handler).
event_manager:delete_handler(eve, stats_handler).
event_manager:add_handler(eve, {stats_handler, OldData}).

This functionally could be added to the event_manager but it would make it less generic as each handler functions differently with get_data and the handler's init state.